tap-gui/plugins/application-accelerator-git-repo-1.5.hbs.md

```diff
@@ -0,0 +1,118 @@
+# Creating an Application Accelerator Git repository during project creation
```

Comment:
As discussed - this change should occur on the original file. No need for the 1.5.0 version of the file.

Reply:
Removed in latest commit